paper/paper.md

```diff
@@ -0,0 +1,183 @@
+  - affiliation: 13
+    name: Alexander P Rockhill
+    orcid: 0000-0003-3868-7453
+  - affiliation: 9
+    name: Eric Larson
+    orcid: 0000-0003-4782-5360
+  - affiliation: 14
+    name: Alexandre Gramfort
+    orcid: 0000-0001-9791-4404
+  - affiliation: 15
+    name: Mainak Jas
+    orcid: 0000-0002-3199-9027
+affiliations:
+- index: 1
+  name: Center for Adaptive Rationality, Max Planck Institute for Human Development, Berlin, Germany
+- index: 2
+  name: Macquarie university
+- index: 3
+  name: Mozilla
+- index: 4
+  name: Department of Neuroscience and Biomedical Engineering, Aalto University, Espoo, Finland
+- index: 5
+  name: Human Cortical Physiology and Neurorehabilitation Section, NINDS, NIH, Bethesda, Maryland 20892
+- index: 6
+  name: UC Berkeley, Project Jupyter
+- index: 7
+  name: Institut du cerveau et de la moelle épinière (ICM), Paris, France
+- index: 8
+  name: Department of Biomedical and Clinical Sciences 'L. Sacco', University of Milan, Milan, Italy
+- index: 9
+  name: Institute for Learning and Brain Sciences, University of Washington, Seattle, WA, USA
+- index: 10
+  name: Institute of Neuroscience and Medicine (INM-3), Research Center Jülich, Germany
+- index: 11
+  name: Max-Planck-Institute for Empirical Aesthetics, Frankfurt a.M., Germany
+- index: 12
+  name: Institute of Psychology, University of Graz, Austria
+- index: 13
+  name: University of Oregon, Eugene OR, USA
+- index: 14
+  name: Université Paris-Saclay, Inria, CEA, Palaiseau, France
+- index: 15
+  name: Athinoula A. Martinos Center for Biomedical Imaging, Massachusetts General Hospital, Charlestown, MA, USA
+date: 29 October 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+With the emergence of the Brain Imaging Data Structure
+(``BIDS``; [@Gorgolewski2016]) in 2016, the neuroscientific community received
+a standard to organize and share data. Originally limited to MRI data formats,
+BIDS is continuously being extended and by now also supports other neuroimaging
+modalities such as MEG [@Niso2018], EEG [@Pernet2019], and iEEG [@Holdgraf2019].
+
+BIDS prescribes how complex experimental data should be structured and which
+metadata should be present besides the raw data. This set of guidelines
+provides the following benefits:
+
+1. **Sharing data within a lab and between labs:** The BIDS specification
+   empowers users to understand any BIDS dataset without requiring specialist
+   knowledge.
+1. **Data validation:** Dedicated software can check whether the rules of
+   BIDS are being followed in a dataset and alert users to missing or broken
+   data (see
+   [bids-validator](https://github.com/bids-standard/bids-validator)).
+1. **Automatic data handling and analysis pipelines:** Knowing which data to
+   expect and where to search for it, many common workflows in data handling
+   and analysis can be simplified or automated through specialized software
+   tools.
+
+To make use of these benefits, datasets must first be converted to the
```

Comment:
I am not sure "conversion" is the right phrase that we want to advertise here. The real power of `mne-bids` is in the ability to interface with MNE seamlessly. BIDS is a complex (yet standardized) data structure that needs specialized software. It's not that `mne-bids` will be useless once all datasets are converted to BIDS format because you still need specialized tools to organize data in BIDS format and to understand them in MNE